README.md

```diff
@@ -1502,6 +1502,7 @@ A [pastebin](https://en.wikipedia.org/wiki/Pastebin) is a type of online content
 - [paaster](https://paaster.io) - Paaster is a secure by default end-to-end encrypted pastebin built with the objective of simplicity. ([Source Code](https://github.com/WardPearce/paaster)) `GPL-3.0` `Docker`
 - [Paste](https://phpaste.sourceforge.io/) - Paste is forked from the original source pastebin.com used before it was bought. ([Source Code](https://github.com/jordansamuel/PASTE)) `GPL-3.0` `PHP`
 - [Pastefy](https://pastefy.ga) - Beautiful, simple and easy to deploy Pastebin with optional Client-Encryption, Multitab-Pastes, an API, a highlighted Editor and more. ([Source Code](https://github.com/interaapps/pastefy), [Clients](https://github.com/topics/pastefy-addon)) `MIT` `Java`
+- [Pasticcio](https://cvcv.cz) - Pastebin with private and public collections. ([Source Code](https://git.sr.ht/~fmac/pasticcio)) `MIT` `Go`
```

Comment:
> https://git.sr.ht/~fmac/pasticcio
> http://ag.fmac.xyz/

Installation docs are WIP, id suggest we leave this until the docs are written

Reply:
Please resubmit when installation documentation is completed.